source/chapters/hardware.rst

```diff
@@ -9,13 +9,34 @@ DJ Hardware
    T.Rafreider <trafreider@mixxx.org>
    S.Brandt <s.brandt@mixxx.org>
    Be <be.0@gmx.com>
+   Jan Holthuis <jholthuis@mixxx.org>
 
 Although Mixxx can be used with just a laptop or desktop computer, fully taking
 advantage of Mixxx's features requires specialized DJ hardware. Depending on
 your budget and application area, your setup and requirements may vary. This
 chapter provides general background information about various types of DJ
 hardware.
 
+To make DJing with Mixxx a fun experience, at least the following pieces are needed:
+
+- speakers to play the music to the audience,
+- an :ref:`audio interface <hardware-audio-interfaces>` with 4 mono/2 stereo
+  output channels (or 2 mono output channels + a :ref:`DJ splitter cable
+  <hardware-splitter-cables>`),
+- :ref:`headphones <hardware-headphones>` for prelistening tracks while another
+  track is playing on the speakers, and
+- audio cables and adapters to connect the equipment to your computer.
+
+Additionally, the follwing equiment will be helpful:
+
```

Comment:
Since the equipment below is optional, I would change this line with

> Additionally, you might consider getting the follwing equiment.

or something like that. Otherwise it sounds like something essential.